sdk/datamigration/Microsoft.Azure.Management.DataMigration/src/Microsoft.Azure.Management.DataMigration.csproj

```diff
@@ -6,10 +6,10 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DataMigration</PackageId>
 		<Description>Provides developers with libraries for the Database Migration Service (DMS) under Azure Resource manager to deploy an instance of DMS. Create, Delete, Read Projects, Files and Tasks and more.</Description>
-		<Version>0.11.0-preview</Version>
+		<Version>0.12.0-preview</Version>
```

Comment:
since we are moving to a stable api-version, should we use a stable version number here?

Reply:
Updated!